src/Microsoft.Framework.FileSystemGlobbing/Matcher.cs

```diff
@@ -10,24 +10,24 @@ namespace Microsoft.Framework.FileSystemGlobbing
 {
     public class Matcher
     {
-        private IList<IPattern> _includePatterns = new List<IPattern>();
-        private IList<IPattern> _excludePatterns = new List<IPattern>();
+        protected IList<IPattern> IncludePatterns { get; set; } = new List<IPattern>();
+        protected IList<IPattern> ExcludePatterns { get; set; } = new List<IPattern>();
```

Comment:
How do you guard against a derived class from adding invalid patterns? Changing a class's design *just* for a unit test is a slippery slope.